Comp2023/src/main/java/frc/robot/Constants.java

```diff
@@ -142,13 +138,28 @@
       ELBOW_UP            // Move elbow up
     }
 
+    public enum ElbowAngle
+    {
+      ELBOW_NOCHANGE,     // No change in elbow Angle--maintain current position
+      ELBOW_STOW, // move arm to stow Angle
+      ELBOW_LOW, // move arm to low-scoring Angle
+      ELBOW_MID, // move arm to shelf Angle; slightly higher than mid-scoring Angle so this is used for both
+      ELBOW_HIGH, // move arm to high-scoring Angle
+    }
+
     public enum WristMode
     {
       WRIST_INIT,         // Initialize wrist
       WRIST_DOWN,         // Move wrist down
       WRIST_STOPPED,      // Stop and hold position
       WRIST_UP            // Move wrist up
     }
+
+    public enum WristAngle
+    {
+      WRIST_NOCHANGE,     // No change in elbow Angle--maintain current position
```

Comment:
Just like the Elbow joint, the wrist will have all of those modes, too, right? As the Elbow moves, the wrist angle will need to be adjusted.

Reply:
fixed